generators/firechip/src/main/scala/TargetMixins.scala

```diff
@@ -72,3 +78,15 @@ trait CanHaveMultiCycleRegfileImp {
   }
 }
 
+trait HasFireSimClockingImp extends HasAdditionalClocks {
+  val outer: HasTiles
+  val (tileClock, tileReset) = p(FireSimClockKey).additionalClocks.headOption match {
```

Comment:
Can you give an example of assigning different clocks to each of the tiles? Is that supported with this right now? It looks like you have the tiles on one domain and the uncore on another.